test/jasmine/tests/plot_api_test.js

```diff
@@ -100,6 +100,35 @@ describe('Test plot api', function() {
             })
             .then(done);
         });
+
+        it('accepts images in layout which it renders', function(done) {
```

Comment:
No need to for this. We already have a full suite of `layout.images` tests in:

https://github.com/plotly/plotly.js/blob/master/test/jasmine/tests/layout_images_test.js

---

We don't currently run any CI test on IE unfortunately, so yeah, your patch won't be supported by any tests at the moment :disappointed: .

We did try to get an Appveyor CI box running (see https://github.com/plotly/plotly.js/issues/431) but that effort came up short.